prov/efa/src/rdm/rxr_pkt_cmd.c

```diff
@@ -798,6 +798,7 @@ void rxr_pkt_handle_send_completion(struct rxr_ep *ep, struct rxr_pkt_entry *pkt
 		rxr_pkt_handle_eor_send_completion(ep, pkt_entry);
 		break;
 	case RXR_RMA_CONTEXT_PKT:
+		// how do we get an RMA Context Packet in a Send completion?
```

Comment:
I added this comment as a reminder to ask others, how do we end up at this case statement?  I want to make sure we handle it correctly in `rxr_pkt_handle_rma_completion` which I made some changes too.